packages/core/src/builtin-tasks/contract.ts

```diff
@@ -213,6 +217,26 @@ subtask('contract:deploy')
 
         writeFileSync(path.join(_migrationPath, migrationFileName), JSON.stringify(migrationData, null, 2))
       }
+
+      const contractManager = new ContractManager([
+        new KuaiContractLoader(config.devNode?.builtInContractConfigPath ?? path.resolve(configPath(), 'scripts.json')),
+      ])
+
+      contractManager.updateContract({
+        name: nameOrBinPath,
+        path: conrtactBinPath,
+        scriptBase: {
+          codeHash: dataHash,
+          hashType: hashType,
+        },
+        outPoint: {
+          txHash: txHash,
+          index: '0x' + index.toString(16),
+        },
+        depType: depType,
+      })
+
+      contractManager.write()
```

Comment:
Use `await` maybe better I think.

Reply:
The "write" function doesn't seem to be async function, will `await` work?

Reply:
> The "write" function doesn't seem to be async function, will `await` work?
It's my fault, I had the wrong understanding before.